katello/katello-backup

```diff
@@ -110,6 +183,11 @@ if @dir.nil?
   puts "**** ERROR: Please specify an export directory ****"
   puts optparse
   exit(-1)
+elsif
+  @options[:snapshot] && (@options[:online] || @options[:incremental])
+  puts "**** ERROR: Snapshot backups can only be done offline ****"
```

Comment:
I was under the impression that the snapshot must be created while services were shut off. If that isn't the case, I'm happy to add an online solution. I'll work on updating this one now.